### PR TITLE
[stable/memcached] Add pod annotations

### PR DESCRIPTION
**Pull request by dkarnutsch**
Signed-off-by: Daniel Karnutsch <d.karnutsch@gmail.com>

#### What this PR does / why we need it:
Add support for pod annotations.

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/memcached/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: memcached
-version: 2.4.0
+version: 2.4.1
```
Comment (olemarkus) [inline stable/memcached/Chart.yaml]: This is a new feature. Could you bump minor version?

**Review by olemarkus — CHANGES_REQUESTED** [review]
Thanks for the PR. Just fix the version and it is good to go

**Comment by dkarnutsch** [comment]
@olemarkus Thanks for your quick review! I've made the changes accordingly.

**Comment by olemarkus** [comment]
Thanks!
/lgtm

**Pull request merged**